apps/pair.py

```diff
@@ -360,10 +360,8 @@ async def pair(
                         await connection.pair()
                     else:
                         await connection.authenticate()
-                    return
```

Comment:
I'd prefer adding a `--linger` option (which could default to `True`) to tell the app to stay connected even after the pairing complete. This way, if a user wants to pair and have the app exit once done, they can still have that behavior.